### DISCUSSION TITLE
The Clock on the waybar shows UTC instead of local time.

### DISCUSSION THREAD

**Discussion by lordherard** [prasanthrangan/hyprdots#1572] [General]
As the title say, I'm starting a fresh install of the Hyprdots, loving it so far but I'm having an issue with the clock showing the UTC instead of the local time and can't find where to switch this config. Any help will be very appreciated.

**Comment by kRHYME7** [comment]
Dual booting? Vanilla arch? Correctly followed arch wiki on setting up locale and local time?  

Provide ` timedatectl ` output.

**Reply by lordherard** [reply]
Hi, single Boot, fresh install, Hyprdots on clean Arch locale set to my zone and local time are correctly showing up on timedatectl, it's the waybar that´s showing the UTC instead of local time.
![240609_18h49m23s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/63799779/d9b4b998-b496-4200-946b-d4d8ed6d9d1a)

**Reply by Olaipai** [reply]
Hey,

I had a similar issue like you.  
I ended up just manually configuring the module clock.jsonc and added  the following:
`"format": "{:%H:%M}",`  
Out of the box the module was set to 12-hour with AM/PM.
